.golangci.yml

```diff
@@ -4,17 +4,23 @@
 # options for analysis running
 run:
   # timeout for analysis, e.g. 30s, 5m, default is 1m
-  timeout: 1m
+  timeout: 15m
```

Comment:
I hope lint doesn't usually take 15 minutes now

Reply:
The mage target uses a template from `dev-tools/templates/.golangci.yml` that differs than the one from the project root. I kept the template one because it was more similar to the Beats one.

But now I'm wondering if it was a good decision... I'll review the template and the overall changes.

Reply:
Ah fair enough. I'm going to remove the templates everywhere, it was a convenience to update the Go version which is now automated.